<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostCheckDialog</class>
 <widget class="QDialog" name="PostCheckDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>319</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Post Checks</string>
  </property>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Number</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkNoEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkAmountEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="checkDateDisplay">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="checkPayeeDisplay">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="checkAmountDisplay">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Payee:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>270</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>checkNoEdit</tabstop>
  <tabstop>checkAmountEdit</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">dao/checkdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
